tracer/src/Datadog.Trace/Iast/IastModule.Escape.cs

```diff
@@ -0,0 +1,59 @@
+// <copyright file="IastModule.Escape.cs" company="Datadog">
+// Unless explicitly stated otherwise all files in this repository are licensed under the Apache 2 License.
+// This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2017 Datadog, Inc.
+// </copyright>
+
+#nullable enable
+
+using System;
+using System.Net;
+using Datadog.Trace.Configuration;
+
+namespace Datadog.Trace.Iast;
+
+internal static partial class IastModule
+{
+    public static string? OnXssEscape(string? text)
+    {
+        var res = WebUtility.HtmlEncode(text);
```

Comment:
Please use the HtmlEncode from System.Net: https://learn.microsoft.com/en-us/dotnet/api/system.net.webutility.htmlencode?view=netframework-4.6.1

This way we don't have to add System.Web to the trimming file.

Reply:
nvm I misread, you're already using the System.Net one